subprojects/docs/src/docs/userguide/running-builds/gradle_daemon.adoc

```diff
@@ -216,11 +216,14 @@ For incremental builds, there are typically only a few changes between builds.
 Therefore, incremental builds can re-use most of the Virtual File System from the last build and benefit the most from watching the file system.
 
 Gradle uses operating system features for watching the file system.
-It supports the feature on these operating systems:
+It supports the feature on these operating systems and file systems:
 
-- Windows 10,
-- Linux (Ubuntu 16.04 or later, CentOS 8 or later, Red Hat Enterprise Linux 8 or later, Amazon Linux 2),
-- macOS 10.14 (Mojave) or later.
+- Windows 10 with NTFS,
+- Linux (Ubuntu 16.04 or later, CentOS 8 or later, Red Hat Enterprise Linux 8 or later, Amazon Linux 2) using ext3 and ext4,
+- macOS 10.14 (Mojave) or later on APFS and HFS+.
+
+Network file systems like NFS and SMB are not supported.
```

Comment:
From this description it is not clear what `not supported` means. I wonder if we should be more precise, and state what you can expect. Maybe that Gradle tries to do the file watching and may fail or not. WDYT?

Reply:
For now SMB simply produces the "cannot watch" error. If we merge #13996 SMB will start to work, but might produce some weirdness (=warnings, crash maybe?) on watcher stop / daemon exit.

I don't know about NFS, as we never tested.

We agreed today that we'd stop investigating further and let the community give feedback. I asked on the community Slack channel about use cases. I'm planning to ask people who reply to try stuff out.

Reply:
Yeah, I don't want us to investigate further. We don't disable file system watching for the above file systems, we let it run and fail. We can leave it at not supported, I was wondering if we could come up with a more precise definition of what not supported means.

Reply:
I think "not supported" is pretty clear for now: you should not use it on those file systems. It will be somewhat more complicated when we turn file system watching on by default.